internal/error.go

```diff
@@ -820,6 +845,24 @@ func (e *WorkflowExecutionError) Unwrap() error {
 	return e.cause
 }
 
+func formatError(e error, s fmt.State, verb rune, verboseValue any) {
```

Comment:
I fear the future proof-ness of this. If Go adds new print flags that'd work for errors, we'd silently not support them.

I wonder if it'd be better to change where you're formatting these errors to handle more advanced types instead of adding a formatting feature to a few of the errors here. I think that gives you the most flexibility instead of assuming exactly what people want when `%+v` is put (you want stack trace, but another might want struct fields). We tend to put accessors on the errors for users to extract and display the way they prefer.

(having said that, I'm not _completely_ against the change, I think it just makes too many assumptions of what users may want to see instead of encouraging them to extract the information themselves when they're displaying)